public/app/plugins/datasource/cloudwatch/datasource.ts

```diff
@@ -123,7 +122,7 @@ export class CloudWatchDatasource
   }
 
   query(options: DataQueryRequest<CloudWatchQuery>): Observable<DataQueryResponse> {
-    options = angular.copy(options);
```

Comment:
👏 is this the last place we are using angular here?

Reply:
yup! 😄

Reply:
cool!